src/Compilers/CSharp/Portable/Symbols/TypeWithAnnotations.cs

```diff
@@ -18,82 +18,15 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     [DebuggerDisplay("{GetDebuggerDisplay(), nq}")]
     internal readonly struct TypeWithAnnotations : IFormattable
     {
-        /// <summary>
-        /// A builder for lazy instances of TypeWithAnnotations.
-        /// </summary>
         [DebuggerDisplay("{GetDebuggerDisplay(), nq}")]
-        internal struct Builder
+        internal sealed class Boxed
```

Comment:
You could extract this into a general purpose type for structs, if you want.

Reply:
I'm holding generalizing the type til we have a second usage.

---
In reply to: [280904409](https://github.com/dotnet/roslyn/pull/35459#discussion_r280904409) [](ancestors = 280904409)